agents/agents-features/agents-features-tokenizer/src/jvmTest/kotlin/ai/koog/agents/features/tokenizer/feature/MessageTokenizerTest.kt

```diff
@@ -0,0 +1,212 @@
+package ai.koog.agents.features.tokenizer.feature
+
+import ai.koog.agents.core.agent.AIAgent
+import ai.koog.agents.core.agent.config.AIAgentConfig
+import ai.koog.agents.core.dsl.builder.forwardTo
+import ai.koog.agents.core.dsl.builder.strategy
+import ai.koog.agents.core.dsl.extension.*
+import ai.koog.agents.core.tools.ToolRegistry
+import ai.koog.agents.testing.feature.withTesting
+import ai.koog.agents.testing.tools.getMockExecutor
+import ai.koog.agents.testing.tools.mockLLMAnswer
+import ai.koog.prompt.dsl.Prompt
+import ai.koog.prompt.dsl.prompt
+import ai.koog.prompt.llm.OllamaModels
+import ai.koog.prompt.message.Message
+import ai.koog.prompt.tokenizer.Tokenizer
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+/**
+ * Test for the MessageTokenizer feature.
+ *
+ * This test verifies that the MessageTokenizer correctly tracks token usage.
+ */
+class MessageTokenizerTest {
```

Comment:
Also, why did you put this test inside `jvmTest`? I don't see it using any platform-specific funcionality, let's move it in `commonMain`?

Reply:
You should use `runTest`, it's multiplatform and is generally preferred in tests
https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/run-test.html. You can check example usage in [MCP tests](https://github.com/JetBrains/koog/blob/0efd4150d4ecdf4c15fefa6f33b0a3b6d1d11946/agents/agents-mcp/src/jvmTest/kotlin/ai/koog/agents/mcp/McpToolTest.kt#L41)